specs/DesignDiscussions/InputLabelsOptionLabel.md

```diff
@@ -0,0 +1,440 @@
+
+##### Non-adjacent labels
+
+Following this approach, non-adjacent labels are not going to be possible as labels are going to be tied to the inputs which will make it easier to users to know what label corresponds to each input.
+
+As this approach sets the rules of how labels are rendered, it also limits where a label can be drawn in valid scenarios as the next ones:
+
+![img](assets/InputLabels/LabelsToTheLeft.png)
+
+I built this card using a series of column sets. Although it is an acceptable UI, this is probably not the approach we want for card authors to label their inputs on the side. The card authoring is fairly non-intuitive to get a good result. If we want to support labels to the left of the input, we should probably do this as a first-class explicit property (```"labelPosition"``` or ```"allowHorizontalLabels"```). 
+
+This card is another possible example of a use case for non-adjacent labels, where there is additional information between the label and the input:
+
+![img](assets/InputLabels/AdditionalLabelContent.png)
+
+This scenario could be covered by a RichTextBlock that contains both pieces of text being used as the label (arguably the entire text should be read as the accessibility label for the input). 
+
+This approach will limit what elements can be rendered as the label to only text.
+
+## Backwards Compatibility
+On earlier versions of renderers the label would be completely dropped, leading to a bad user experience. The card may be completely unusable if without the labels the user cannot deduce what information is meant to be provided in the unlabeled inputs.
+
+Because the label is a property, not a new type, card authors would not be able to use type to trigger fallback to an independent text block and input. To support this, the card author would need to use requires as follows:
+
+```json
+{
+	"type": "Input.Text",
+	"id": "NameId",
+	"label": "Please enter your name",
+	"requires": {
+		"adaptiveCards": "1.3"
+	},
+	"fallback": {
+		"type": "Container",
+		"items": [
+			{
+				"type": "TextBlock",
+				"text": "Please enter your name:"
+			},
+			{
+				"type": "Input.Text",
+				"id": "NameId"
+			}
+		]
+    }
+}
+```
+We could also possibly add a specific named feature to the requires dictionary to allow card authors to address it directly using something like:
+```json
+"requires": {
+	"inputLabels": "1.0"
+}
+```
+Either of these approaches would allow card authors who know their cards will be hitting platforms targeting both 1.2 and 1.3 to take advantage of the labels for accessibility or interactivity purposes in 1.3 without breaking their experience on 1.2 clients. That this will not help card authors who are targeting 1.0 or 1.1 hosts where fallback is not supported. Card authors who know their hosts support version 1.3 (or any future version) will not need to worry about this concern.
+
+## Placeholders
```

Comment:
Great. Also arent we getting guidance from the Accessibility team that Placeholders in general arent that useful? 
<snip>
While placeholders have historically been very common in apps, there’s a growing view these days that from a usability perspective, placeholders should be avoided. At the end of Sarah’s [What's in a name?](https://sarahmhigley.com/writing/whats-in-a-name/#tooltips%2C-placeholders%2C-names%2C-and-duplicate-announcements) article, she references a number of articles which discuss why placeholders shouldn’t be used. Some of the content relates to web UI, but the principles around design apply to any type of UI.

So you may want to review some of the articles, such as [Placeholders in Form Fields Are Harmful](https://www.nngroup.com/articles/form-design-placeholders/), and consider if perhaps your customers would be better helped by having (as the article says) “clear, visible labels that are placed outside empty form fields.”
</snip>